tests/acceptance/expected-failures-webUI-on-OCIS-storage.md

```diff
@@ -318,7 +314,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:236](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L236)
 
 ### [renamed file cannot be shared](https://github.com/owncloud/web/issues/4192)
--   [webUISharingInternalUsers/shareWithUsers.feature:341](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L341)
```

Comment:
If this is the only failure for issue 4192, then the heading could be deleted also.